composer/core/evaluator.py

```diff
@@ -22,10 +22,11 @@ def evaluate_periodically(eval_interval: Union[str, Time, int]):
     """Helper function to generate an evaluation interval callable.
 
     Args:
-        eval_interval (str | Time | int): A :class:`.Time` instance or time string, or integer in epochs,
-            representing how often to evaluate. Set to ``0`` to disable evaluation.
+        eval_interval (str | :class:`.Time` | int): A :class:`.Time` instance, time string, 
```

Comment:
The `Time` was already autolinking, do we need the class updated here?

Reply:
Fixed